content/sensu-go/5.21/reference/agent.md

```diff
@@ -715,7 +715,15 @@ Specify the agent configuration with either a `.yml` file or `sensu-agent start`
 Configuration via command line flags overrides attributes specified in a configuration file.
 See the [Example Sensu agent configuration file][5] for flags and defaults.
 
-#### Configuration summary
+### Certificate bundles or chains
+
+The Sensu backend supports all types of certificate bundles (or chains) as long as the server (or leaf) certificate is the *first* certificate in the bundle.
```

Comment:
I'm assuming this should mention "the sensu agent" instead of "the sensu backend" given that it's the agent reference?

Reply:
ditto with "the server (or leaf) certificate"; I think that should read "the agent (or leaf) certificate".